File: client/Droid/MyApplication.cs
using System;
using Android.App;
using Com.Videogo.Openapi;

[assembly: Application(
    Debuggable = true,
    Label = "SmartConstructionSite",
    ManageSpaceActivity = typeof(SmartConstructionSite.Droid.MyApplication))]
namespace SmartConstructionSite.Droid
{
    public class MyApplication : Application
    {
        const string AppKey = "443eed7d6dab47739915d6a237dcad34";

        public MyApplication()
        {
        }

        public override void OnCreate()
        {
            base.OnCreate();
            InitSDK();
        }

        private void InitSDK()
        {
            /**
             * sdk日志开关，正式发布需要去掉
             */
            EZOpenSDK.ShowSDKLog(true);

            /**
             * 设置是否支持P2P取流,详见api
             */
            EZOpenSDK.EnableP2P(true);

            /**
             * APP_KEY请替换成自己申请的
             */
            EZOpenSDK.InitLib(this, AppKey, "");
        }
    }
}

File: client2/cordova/gdzs_cordova_plugin_cameras/app/src/main/java/cn/yooling/app/gdzs/plugin/MyApplication.java
package cn.yooling.app.gdzs.plugin;

import android.app.Application;

import com.ezvizuikit.open.EZUIKit;
import com.videogo.openapi.EZOpenSDK;

public class MyApplication extends Application {
    public static final String APP_KEY = "443eed7d6dab47739915d6a237dcad34";

    @Override
    public void onCreate() {
        super.onCreate();
        initSDK();
    }

    private void initSDK() {
        {
            /**
             * sdk日志开关，正式发布需要去掉
             */
            EZOpenSDK.showSDKLog(true);

            /**
             * 设置是否支持P2P取流,详见api
             */
            EZOpenSDK.enableP2P(true);

            /**
             * APP_KEY请替换成自己申请的
             */
            EZOpenSDK.initLib(this, APP_KEY);
        }
    }
}
